### DISCUSSION TITLE
专业版升级问题

### DISCUSSION THREAD

**Discussion by Valdstejna** [getlantern/lantern-forum-cn#815] [Q&A]
8月6日续费两年后并没有升级到专业版，之前通过Lantern Helpdesk发问题报告只有公式化回答。
邮箱是kszz0818562@gmail.com，helpdesc上的地址是https://lantern.jitbit.com/helpdesk/Ticket/76166982
缴费凭证在helpdesc上已上传

**Comment by MaxMKMax** [comment]
@Valdstejna 您好. 好的我已经帮您提交问题上去了. 谢谢您支持蓝灯!
